coalib/parsing/filters/Filter.py

```diff
@@ -0,0 +1,19 @@
+from functools import wraps
+
+
+def filter(filter_applier):
+    def filter_section_bears(bears, args):
```

Comment:
you should rename the `bears` parameter into something that does fit more. It contains also section names as keys^^